File: Java/Car.java
class Car{
    Integer id;
    String license;
    Account driver;
    //private Integer passenger;
    protected Integer passenger;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }

    public Integer getPassenger() {
        return passenger;
    }

    public void setPassenger(Integer passenger) {
        if (passenger == 4){
            this.passenger = passenger;
        } else {
            System.out.println("Necesita asignar 4 pasajeros");
        }
    }

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }

    void printDataCar(){
        if (passenger != null){
            System.out.println("License:" + license + " Driver: " + driver.name +  " Passengers: " + passenger);
        }
        
    }



    

}

File: PHP/Car.php
<?php
require_once('Account.php');
class Car {
    public $id;
    public $license;
    public $driver;
    public $passengers;

    public function getPassenger() {
        return $this->passengers;
    }

    public function setPassenger($passengers) {
        
        if ($passengers == 4) {
            $this->passengers = $passengers;
        }
        else {
            echo "Necesitas asignar 4 pasajeros";
        }
    
    }

    public function __construct($license, $driver){
        $this->license = $license;
        $this->driver =$driver;
    }

    /* public function printDataCar(){
        echo "License: $this->license, Driver:". $this->driver->name;
    } */

    public function printDataCar() {
        echo "
            License: $this->license 
            Driver: {$this->driver->name} 
            Número de pasajeros: $this->passengers ";
    }


}
?>